cmd/main.go

```diff
@@ -42,6 +43,8 @@ func main() {
 		return
 	}
 
+	spverrors.SetupGlobalZerologErrorHandler()
```

Comment:
shouldn't it be a part of logging package?
and why it isn't placed just after default logger (or even earlier)

Reply:
Yes and yes 👍

Reply:
Done :)